subsys/logging/Kconfig.backends

```diff
@@ -3,6 +3,12 @@
 
 menu "Backends"
 
+config LOG_BACKEND_EVENT_NOTIFICATION_SUPPORT
+	bool "Support Backend Event Notifications"
+	help
+	  This allows backend to be notified of events
```

Comment:
maybe example of an event could be mentioned here (example is the only one currently supported)

Reply:
This kconfig is removed since with the new changes to remove `#ifdef`, it doesn't make sense to have this kconfig anymore.